samcli/lib/package/ecr_uploader.py

```diff
@@ -76,10 +77,26 @@ def upload(self, image, resource_name):
         if not self.login_session_active:
             self.login()
             self.login_session_active = True
+
+        # cosborn: We sometimes use the resource_name as the `image` parameter to `tag_translation`.
```

Comment:
What does cosborn mean here?

Reply:
That's me. Around the code base, larger comments are signed, so I tried to follow suit. I'm happy to remove it if there's a convention.

Reply:
I think for the final merge, we don't need to the signature anymore, otherwise the documentation is very clear, maybe would prefer if it less personal instead of using terms like We but thats just me nitpicking

Reply:
Sure thing. I'll make that change now.